app/helpers/votes_helper.rb

```diff
@@ -0,0 +1,8 @@
+module VotesHelper
+  # check it for 'vote', made by current_user
+  def voted(votes)
```

Comment:
This method is a predicate, it asks a specific question and can return a boolean `true` or `false`. 
In Ruby, methods like this are usually named with a question mark. 

So this method has to be named something like `current_user_voted?`